### ISSUE TITLE
react app error  when load wcjs-player

### ISSUE THREAD

**Issue by mas1515** [jaruba/wcjs-player#91]
hello @jaruba 

 i want load wcjs-player
in react app   but 
it give me this error 

https://pasteboard.co/J9RAu7i.png

my code is this image 

https://pasteboard.co/J9RBi6f.png

please help me .

**Comment by jaruba** [comment]
I wouldn't really recommend using `wcjs` anymore as it was discontinued and relies on VLC 2.2 and probably older Electron versions too.

But if you insist on using it, then this is the relevant explanation regarding the error you get: https://github.com/jaruba/wcjs-player/issues/88

**Comment by mas1515** [comment]
@jaruba 

can you make simple react js web app demo ???

**Comment by mas1515** [comment]
@jaruba 

another question 
is it possible   use   wcjs-player

in simple  html page without nodejs .

**Comment by jaruba** [comment]
> I am not sure what should I look for on these screenshots.

no, i can't dedicate time to making a react example for you as `wcjs` has been discontinued. unless it gets restarted, i cannot dedicate more time to examples for it

> is it possible use wcjs-player in simple html page without nodejs .

no, `wcjs` is nodejs only

**Comment by mas1515** [comment]
@jaruba 
thanks for reply .
What do you think I should use instead of webchimera ?
I want   play video with rtsp address
To show in  reacjs app ,

**Comment by RSATom** [comment]
@mas1515 do you need to do it in browser? if yes - one possible solution will be use some server to convert rtsp to WebRTC.

**Comment by mas1515** [comment]
@RSATom   
thanks for reply 

 
I do not know  webrtc very good 
I need a regular player
that play  rstp on web 
Like ip camera
rtsp: // admin: test@192.168.1.64: 554 / cam / realmonitor? channel = 1 & subtype = 0

what is your resolve ??
for simple web html page and nodejs  and react js

i test video.js and jwplayer but not support rtsp ,

**Comment by mas1515** [comment]
@RSATom 
another question 
can you send simple react js  app that use webchimera
not electron . just react js
thanks

**Comment by RSATom** [comment]
Browsers can't support rtsp by internal restrictions, so I don't see any way to play rtsp in browser without transcoding...
What about demo app for webchimera - I will not spend time on it while it in deprecated state, I have better ways to use my free time.
